test/runtests.jl

```diff
@@ -10,7 +10,7 @@ import Random.seed!
 using Statistics
 import StatsBase
 using StableRNGs
-using cuDNN
```

Comment:
@ablaom is this ok? I didn't see it used anywhere, should I just remove it?

Reply:
I think this was the reason for tests timing out before.

Reply:
I suggest we open an issue to track why this happens but without blocking this PR.

Reply:
https://github.com/FluxML/Flux.jl/issues/2346

Reply:
should be fixed now by adding CUDA as well as a test dep